samples/bluetooth/hci_usb_h4/README.rst

```diff
@@ -0,0 +1,23 @@
+.. _bluetooth-hci-usb-h4-sample:
+
+Bluetooth: HCI H4 over USB
+##########################
+
+Overview
+********
+
+Make a USB H4 Bluetooth dongle out of Zephyr. Requires USB device support from
+the board it runs on (e.g. :ref:`nrf52840_pca10056` supports both BLE and USB).
+
+Requirements
+************
+
+* Bluetooth stack running on the host (e.g. BlueZ)
```

Comment:
Does it depend on btusb driver/kernel-version?

Reply:
It does, @holtmann will be working on that.

Reply:
okay, please let usb people on the slack know when the btusb patches are coming.

Reply:
Sure, I will update this series when we are done with the btusb changes, we do have an out of the tree driver that works already.

Reply:
Yes there is the out of tree driver the @holtmann created:

https://github.com/holtmann/zpusb

Reply:
I get "Page not found" on that link. But not important. :)

Reply:
Perhaps the tree is still not public, need to check with @holtmann